pkgs/games/gl-gsync-demo/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, libX11, freeglut, glew, libXNVCtrl, libXext, lib }:
+
+let
+  pname = "gl-gsync-demo";
+
+  version = "2020-12-27";
+
+  src = fetchFromGitHub {
+    owner = "dahenry";
+    repo = "gl-gsync-demo";
+    rev = "4fd963a8ad880dc2d846394c8c80b2091a119591";
+    sha256 = "1innsmpsd9n9ih80v16rhj2ijrl28jd7x6a4jmxdirba7mjibm8d";
+  };
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  buildInputs = [ libX11.dev freeglut.dev glew.dev libXNVCtrl libXext ];
```

```suggestion
  buildInputs = [ libX11 freeglut glew libXNVCtrl libXext ];
```

Nix is smart enough to detect this.